manifests/r/RoyalApps/RoyalTS/6.1.60512.0/RoyalApps.RoyalTS.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: RoyalApps.RoyalTS
+PackageVersion: 6.1.60512.0
+PackageLocale: en-US
+Publisher: Royal Apps GmbH
+PackageName: Royal TS
+PackageUrl: https://www.royalapps.com/ts/win/features
+License: Copyright 2021 Royal Apps GmbH

```

```suggestion
License: Proprietary
```
A copyright isn't a license